resolvers.ts

```diff
@@ -945,11 +938,24 @@ export const resolvers: Resolvers = {
       // TODO (nina): call nextgen in addition to rails to get CG count
     },
     ZipLink: async (root, args, context, info) => {
-      const res = await getFullResponse(
-        `/workflow_runs/${args.workflowRunId}/zip_link.json`,
+      // const nextGenEnabled = await shouldReadFromNextGen(context);
```

Comment:
do we still need to keep this comment?

Reply:
I am keeping it here for now as an example for other devs of how to use the custom query field. I will remove it when I edit Ziplink. Does that sound okay?